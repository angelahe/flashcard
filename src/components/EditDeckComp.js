import React from 'react';
import '../styles/flashcard.css';
import addbtn from '../img/buttons/add_FFFFFF.png';

class EditDeckComp extends React.Component {
  constructor() {
    super();
    this.state = {
      currentDeck: '',
      deckList: [],
    };
  }

  render() {
    return (
      <div>
        <p>EditDeckComp</p>
      </div>
    );
  }
}
export default EditDeckComp;
