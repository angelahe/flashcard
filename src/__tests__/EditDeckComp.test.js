import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import EditDeckComp from '../components/EditDeckComp';

it('renders without crashing', () => {
  shallow(<EditDeckComp />);
});

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<EditDeckComp />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test('snapshot of UI renders consistently', () => {
  const tree = renderer.create(<EditDeckComp />).toJSON();
  expect(tree).toMatchSnapshot();
});
