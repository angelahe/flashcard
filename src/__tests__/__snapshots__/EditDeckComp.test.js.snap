// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of UI renders consistently 1`] = `
<div
  className="EditDeckComp"
>
  <p>
    EditDeckComp is coming soon
  </p>
</div>
`;
